app/views/proprietor/accounts/_shared_search_toggle.html.erb

```diff
@@ -0,0 +1,71 @@
+
+<!-- Originally from https://github.com/samvera/hyrax/blob/main/app/views/hyrax/admin/features/index.html.erb -->
+<% feature_set = all_features  %>
+<% grouped_features = feature_for_display_in_proprietor_account_ui %>
+
+<div class="flip row">
+  <div class="col-md-12">
+    <div class="panel">
+      <div class="panel-body">
+        <div class="table-responsive">
+          <table class="table table-striped">
+            <thead>
+              <tr>
+                <th></th>
+                <th class="name"><%= t('.feature') %></th>
+                <th class="description"><%= t('.description') %></th>
+                <th class="action"><%= t('.action') %></th>
+              </tr>
+            </thead>
+            <tbody>
+            <%# grouped_features&.each do |group, features| -%>
+            <% grouped_features&.each do |feature| -%>
+
+              <% if feature_set.grouped? -%>
+                <tr class="group">
+                  <td></td>
+                  <td class="name" colspan="<%= 2 + feature_set.strategies.size -%>">
+                    <h2>
+                      <%# t(group ? group.name : :default, scope: [:flipflop, :groups], default: group ? group.title : nil) -%>
+                    </h2>
+                  </td>
+                </tr>
+              <% end -%>
+              <%# features&.each do |feature| %>
+              <tr data-feature="<%= feature.name.dasherize.parameterize %>">
+                <td class="status">
+                  <span class="<%= feature_set.status(feature) -%>"><%= feature_set.status(feature) -%></span>
+                </td>
+                <td class="name"><%= feature.name.humanize -%></td>
+                <td class="description"><%= feature.description -%></td>
+
+                <% feature_set.strategies.each do |strategy| -%>
+                  <% next unless strategy.is_a? Flipflop::Strategies::ActiveRecordStrategy %>
```

Comment:
maybe this line and its top one should go into a helper method like:
```
def per_valid_feature_set_strategy(feature_set)
    feature_set.strategies.each do |strategy|
       yield(strategy) if strategy.is_a? Flipflop::Strategies::ActiveRecordStrategy
    end 
end
```

Reply:
let's keep the file as it came from hyrax rather than improve it.